This is a hack for which I have found no alternative; it allows
gobject-introspection to find the nss/nspr libraries when it's scanning the
libraries.  Since it isn't used by any binaries or libraries we deliver, it
should be ok, but I wish there was a better solution.

Not appropriate for upstream.
--- evolution-data-server-3.16.2/addressbook/libebook/Makefile.am	2015-03-22 23:41:04.000000000 -0700
+++ evolution-data-server-3.16.2/addressbook/libebook/Makefile.am	2015-07-16 18:27:14.254242705 -0700
@@ -133,6 +133,10 @@
 	$(top_builddir)/addressbook/libedata-book/libedata-book-1.2.la \
 	libebook-1.2.la $(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(NULL)
+EBook_1_2_gir_LDFLAGS = \
+	-L/usr/lib/mps/64 \
+	-L/usr/lib/mps \
+	$(NULL)
 EBook_1_2_gir_FILES = \
 	$(libebook_1_2_la_SOURCES) \
 	$(libebookinclude_HEADERS) \
--- evolution-data-server-3.16.2/addressbook/libebook-contacts/Makefile.am	2015-03-22 23:41:04.000000000 -0700
+++ evolution-data-server-3.16.2/addressbook/libebook-contacts/Makefile.am	2015-07-16 18:27:21.249507947 -0700
@@ -142,6 +142,10 @@
 EBookContacts_1_2_gir_LIBS = \
 	libebook-contacts-1.2.la $(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(NULL)
+EBookContacts_1_2_gir_LDFLAGS = \
+	-L/usr/lib/mps/64 \
+	-L/usr/lib/mps \
+	$(NULL)
 EBookContacts_1_2_gir_FILES = \
 	$(libebook_contacts_1_2_la_SOURCES) \
 	$(libebookcontactsinclude_HEADERS) \
--- evolution-data-server-3.16.2/libedataserver/Makefile.am	2015-03-22 23:41:04.000000000 -0700
+++ evolution-data-server-3.16.2/libedataserver/Makefile.am	2015-07-16 18:27:44.552846725 -0700
@@ -261,6 +261,10 @@
 	libedataserver-1.2.la \
 	$(top_builddir)/camel/libcamel-1.2.la \
 	$(NULL)
+EDataServer_1_2_gir_LDFLAGS = \
+	-L/usr/lib/mps/64 \
+	-L/usr/lib/mps \
+	$(NULL)
 EDataServer_1_2_gir_FILES = $(introspection_sources)
 INTROSPECTION_GIRS += EDataServer-1.2.gir
